overlord/snapstate/autorefresh.go

```diff
@@ -599,7 +599,7 @@ var asyncPendingRefreshNotification = func(context context.Context, client *user
 // Internally the snap state is updated to remember when the inhibition first
 // took place. Apps can inhibit refreshes for up to "maxInhibition", beyond
 // that period the refresh will go ahead despite application activity.
-func inhibitRefresh(st *state.State, snapst *SnapState, info *snap.Info, checker func(*snap.Info) error) error {
+func inhibitRefresh(st *state.State, snapsup *SnapSetup, snapst *SnapState, info *snap.Info, checker func(*snap.Info) error) error {
```

Comment:
nitpick: it seems generally when we have both we sort them snapst, snapsup like you did in refresh.go

Reply:
Ah yes, I reverted the order of the others to be consistent and overlooked this one. Fixed it now